smriprep/workflows/outputs.py

```diff
@@ -296,33 +267,33 @@ def init_anat_derivatives_wf(*, bids_root, freesurfer, num_t1w, output_dir,
         (t1w_name, ds_t1w_dseg, [('out', 'source_file')]),
         (t1w_name, ds_t1w_tpms, [('out', 'source_file')]),
         (raw_sources, ds_t1w_mask, [('out', 'RawSources')]),
-        # Template
-        (inputnode, ds_t1w_tpl_warp, [
-            ('anat2std_xfm', 'in_file'),
-            (('template', _drop_cohort), 'to')]),
-        (inputnode, ds_t1w_tpl_inv_warp, [
-            ('std2anat_xfm', 'in_file'),
-            (('template', _drop_cohort), 'from')]),
-        (inputnode, ds_t1w_tpl, [
-            ('std_t1w', 'in_file'),
-            (('template', _fmt_cohort), 'space')]),
-        (inputnode, ds_std_mask, [
-            ('std_mask', 'in_file'),
-            (('template', _fmt_cohort), 'space'),
-            (('template', _rawsources), 'RawSources')]),
-        (inputnode, ds_std_dseg, [(('template', _fmt_cohort), 'space')]),
-        (inputnode, ds_std_dseg, [('std_dseg', 'in_file')]),
-        (inputnode, ds_std_tpms, [
-            ('std_tpms', 'in_file'),
-            (('template', _fmt_cohort), 'space')]),
-        (t1w_name, ds_t1w_tpl_warp, [('out', 'source_file')]),
-        (t1w_name, ds_t1w_tpl_inv_warp, [('out', 'source_file')]),
-        (t1w_name, ds_t1w_tpl, [('out', 'source_file')]),
-        (t1w_name, ds_std_mask, [('out', 'source_file')]),
-        (t1w_name, ds_std_dseg, [('out', 'source_file')]),
-        (t1w_name, ds_std_tpms, [('out', 'source_file')]),
     ])
 
+    # Transforms
+    if spaces.get_spaces(nonstandard=False, dim=(3,)):
```

Comment:
this branch checks whether any standard space was selected at all. If not, there won't be any transforms to write out here.